### PR TITLE
new layer ch.swisstopo.geologie-generalkarte-ggk200.metadata

### PR DESCRIPTION
**Pull request by ltclm**

### CODE REVIEW

**Comment by alcapat** [comment]
when this PR is merged, please contact @ltclm , need to set staging to prod in bod.dataset

**Comment by loicgasser** [comment]
That is why Jenkins is complaining for the hmlpopup I guess.

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -697,6 +697,23 @@ class GeolGenKarteGGK200(Base, Vector):
 register('ch.swisstopo.geologie-generalkarte-ggk200', GeolGenKarteGGK200)
 
 
+class GeolGenKarteGGK200Meta(Base, Vector):
+    __tablename__ = 'kv_ggk_pk'
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: same table as ch.swisstopo.geologie-generalkarte-ggk200.metadata?

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -3609,6 +3609,25 @@ msgstr "Maximum dal temp da glatsch"
 msgid "ch.swisstopo.geologie-generalkarte-ggk200"
 msgstr "Charta geologica generala"
 
+msgid "ch.swisstopo.geologie-generalkarte-ggk200.metadata"
+msgstr "Einteilung geol. Generalkarte 200"
+
+#. TODO
+msgid "ch.swisstopo.geologie-generalkarte-ggk200.metadata.autor"
+msgstr "ch.swisstopo.geologie-generalkarte-ggk200.metadata.autor"
```
Comment (loicgasser) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: translations are missing in rm. Put german instead.

File: chsdi/models/vector/stopo.py (hunk 1 above)
Comment (ltclm) [inline chsdi/models/vector/stopo.py]: yes, once for the raster layer tooltip and once for the vector (meta) layer tooltip.

**Comment by loicgasser** [comment]
jenkins test

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po (hunk 2 above)
Comment (ltclm) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: rm has been added

**Comment by loicgasser** [comment]
jenkins fails because of ch.swisstopo.swissimage-product that was just merged, so ok for me.

**Pull request merged**